test/runtime/monitor.go

```diff
@@ -87,117 +100,117 @@ var _ = Describe("RuntimeMonitorTest", func() {
 			"capture": "DEBUG:",
 		}
 
-		res := cilium.Exec("config Debug=true DropNotification=true TraceNotification=true")
+		res := cilium.Exec(fmt.Sprintf("config %s=true %s=true %s=true", MonitorDebug, MonitorDropNotification, MonitorTraceNotification))
```

Comment:
Could you separate this line into 2 lines?

Reply:
Done.